tools/dreport.d/plugins.d/bmcweb

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Need BMCweb maintainer approval before merging this into ghe.

Reply:
@sunharis @gtmills Please check on this. This was the additional sessions & event data that was agreed to be captured as part of BMC dump for downstream. bmcweb changes for this was merged a while back.